src/styles/_usage-graph.scss

```diff
@@ -20,56 +21,56 @@
     color: var(--pf-global--Color--100);
   }
 
-  .curiosity-stack-chart-container svg {
-    > g:nth-child(2) {
-      > g {
-        display: none;
-      }
+  // .curiosity-stack-chart-container svg {
+  //   > g:nth-child(2) {
+  //     > g {
+  //       display: none;
+  //     }
```

Comment:
clean up: anything commented out should be removed... since it's source controlled we'll pull it back out if needed 😄

Reply:
removed